wiki/configuration/3.7/api.html

```diff
@@ -1175,12 +1179,27 @@ <h2 id="showlog">Get the anonymized log file</h2>
 <p><span class="label label-warning">NOTE</span> This call ignores the <code>output</code> parameter, it will always serve the file as a download.</p>
 <pre><code>api?mode=showlog</code></pre>
 
+<h2 id="del_config">Delete config item(s) within limited sections</h2>
```

Comment:
Header should be the same as the link in the table above. Should both be something like "Delete configuration item within "servers", "rss", or "categories" sections".

Reply:
the href/id do match. but i belive your saying the header text and the table text souldd be the same? i noticed some are and some arent..  ill update this one to match. and then add to 3.5/3.6 as well